.github/workflows/pull_request.yml

```diff
@@ -179,6 +179,17 @@ jobs:
           # shellcheck disable=SC2046
           docker rm -f $(docker ps -a -q) ||:
           sudo rm -fr "$TEMP_PATH" "$CACHES_PATH"
+  SimpleCheck:
+    needs: [StyleCheck, FastTest]
+    runs-on: ubuntu-latest
+    if: ${{ always() }}
+    steps:
+      - name: Allow simple fail
+        if: ${{ contains(github.event.pull_request.labels.*.name, 'allow simple fail') }}
```

Comment:
The label should be named in a more neutral way.
Not "allow fail" but "skip check".